src/Reflex/Dom/Pandoc/Document.hs

```diff
@@ -29,6 +29,7 @@ import Control.Monad.Reader
 import Data.Bool
 import qualified Data.Map as Map
 import Data.Maybe
+import qualified Data.Text as T (pack)
```

Comment:
Why `(pack)`?

Reply:
Old typescript habit from doing explicit imports that has apparently unconsciously infected my haskell programming. I can fix that :)